GVFS/GVFS.Common/Git/GitAuthentication.cs

```diff
@@ -66,15 +66,20 @@ public void ApproveCredentials(ITracer tracer, string credentialString)
                         string password;
                         if (TryParseCredentialString(this.cachedCredentialString, out username, out password))
                         {
-                            this.git.ApproveCredentials(this.repoUrl, username, password);
+                            if (!this.credentialStore.TryStoreCredential(tracer, this.repoUrl, username, password, out string error))
+                            {
+                                // Storing credentials is best effort attempt - log failure, but do not fail
+                                tracer.RelatedWarning("Failed to store credential string: {0}", error);
+                            }
+
                             this.isCachedCredentialStringApproved = true;
                         }
                         else
                         {
-                            EventMetadata metadata = new EventMetadata(new Dictionary<string, object>
+                            EventMetadata metadata = new EventMetadata(new Dictionary<string, object>
                             {
-                                ["RepoUrl"] = this.repoUrl,
-                                ["CredentialString"] = this.cachedCredentialString
+                                ["RepoUrl"] = this.repoUrl,
+                                ["CredentialString"] = this.cachedCredentialString
                             });
                             tracer.RelatedError(metadata, "Failed to parse credential string for approval");

```

Comment:
The variable name `CredentialString` suggests it is holding sensitive information. I know we were doing this before, is it encrypted? & okay to send it to LiveSite?

Reply:
Will remove